### PR TITLE
Jbrowse: fix errors in singularity

### PR DESCRIPTION
**Pull request by abretaud**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [x] - This PR does something else (explain below)

Fixing the errors from #2701

### CODE REVIEW

**Comment by mvdbeek** [comment]
👍 FWIW https://github.com/galaxyproject/galaxy/pull/9088 would fix this, or we can try source /usr/local/etc/conda/activate.d scripts when building command lines for biocontainers.

File: tools/jbrowse/jbrowse-fromdir.xml [hunk 1]
```diff
@@ -7,12 +7,15 @@
   <command detect_errors="exit_code"><![CDATA[
 ## Create output directory
 mkdir -p $output.files_path &&
+
+export JBROWSE_SOURCE_DIR=\$(dirname \$(which prepare-refseqs.pl))/../opt/jbrowse  &&
```
Comment (mvdbeek) [inline tools/jbrowse/jbrowse-fromdir.xml]:
```suggestion
export JBROWSE_SOURCE_DIR=\$(dirname \$(command -v prepare-refseqs.pl))/../opt/jbrowse  &&
```
Should be more portable (not that it really matters IMO)

**Comment by abretaud** [comment]
Cool if galaxyproject/galaxy#9088 fixes all these activate.d loading!
I guess it makes no harm to redefine the env var here anyway

**Comment by mvdbeek** [comment]
Yeah, I don't think we've got a consensus yet if we want this and how to do it, but this would definitely work.

**Pull request merged**